File: source/kCura.Relativity.Export.ConvertedToC#/Exceptions/ExportBaseException.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Exceptions
{
	public abstract class ExportBaseException : System.Exception
	{
		protected ExportBaseException(string message, System.Exception innerException) : base(message, innerException)
		{
		}
	}
}

File: source/kCura.Relativity.Export/Exceptions/ExportBaseException.vb
Namespace kCura.Relativity.Export.Exceptions
	Public MustInherit Class ExportBaseException
		Inherits System.Exception
		Protected Sub New(ByVal message As String, ByVal innerException As System.Exception)
			MyBase.new(message, innerException)
		End Sub
	End Class
End Namespace